src/main/java/org/opentripplanner/middleware/models/OtpUser.java

```diff
@@ -78,4 +80,18 @@ public boolean delete(boolean deleteAuth0User) {
 
         return Persistence.otpUsers.removeById(this.id);
     }
+
+    /**
+     * Confirm that the requesting user has the required permissions
+     */
+    @Override
+    public boolean canBeManagedBy(RequestingUser requestingUser) {
+        if (requestingUser.apiUser != null && requestingUser.apiUser.id.equals(applicationId)) {
```

Comment:
Aforementioned isThirdPartyUser method.

Reply:
Could this user be created by both first and third party api users? If that is the case the use of isThirdPartyUser would discount first party users.

Reply:
I don't think we're expecting first party API users to create OtpUsers. That said, there are still quite a few checks for `requestingUser.apiUser != null` in this branch that may need to be re-evaluated for replacing with `isThirdPartyUser`... for now, please just create an issue to document this so we can handle outside of this PR.

Reply:
Issue created: https://github.com/ibi-group/otp-middleware/issues/98